packages/devextreme/playground/jquery.html

```diff
@@ -52,9 +52,12 @@ <h1 style="position: fixed; left: 0; top: 0; clip: rect(1px, 1px, 1px, 1px);">Te
         <div id="button"></div>
         <script>
             $(function() {
-                $("#button").dxButton({
-                    text: 'Click me!',
-                    onClick: function() { alert("clicked"); }
+                $("#button").dxChat({
+                    typingUsers: [
```

Comment:
Let's revert jquery.html

Reply:
I haven't yet given the PR to review after the last edits.

Reply:
[1f65378087cbfdce9005809e7878398685034413]